sapl/comissoes/views.py

```diff
@@ -162,25 +165,28 @@ def form_valid(self, form):
             return super(Crud.UpdateView, self).form_valid(form)
 
 
+def lista_materias_comissao(comissao_pk):
+    ts = Tramitacao.objects.order_by(
+        'materia', '-data_tramitacao', '-id').annotate(
+        comissao=F('unidade_tramitacao_destino__comissao')).distinct(
+            'materia').values_list('materia', 'comissao')
+
+    ts = list(filter(lambda x: x[1] == int(comissao_pk), ts))
```

Comment:
Eu acredito que usando list comprehensions a legibilidade de código fica melhor. 
```suggestion
    ts = [m for (m, c) in ts if c == int(comissao_pk)]
```

Isso substitui as linhas 174 a 176! :)